### PR TITLE
adjustments sidebar and alter bg color

### PR DESCRIPTION
**Pull request by mbr4nt**
leave open

### CODE REVIEW

File: hni-textiles-facets.html [hunk 1]
```diff
@@ -35,6 +35,7 @@
         
         
         paper-input {
+            background:url(/components/hni-textiles-webcomponents/images/filters-bg-top.jpg) no-repeat right top;
```
Comment (mbr4nt) [inline hni-textiles-facets.html]: @felipebbtoledo path should be relative to the component, so just url("images/filters-bg-top.jpg")

File: hni-textiles-search-page.html [hunk 2]
```diff
@@ -1,8 +1,17 @@
 <dom-module id="hni-textiles-search-page">
     <style>
         .filters {
-            width: 25%;
-            float: left;
+            /*width: 25%;
+            position:absolute;
+            left:0;
+            top:0;*/
+            background:url(/components/hni-textiles-webcomponents/images/filters-bg.gif) repeat-y right center;
```
Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo same here

File: hni-textiles-search-result-family.html [hunk 3]
```diff
@@ -28,10 +28,14 @@
         poly-grid {
             margin:-10px;
         }
+        .alter-bg-color {
+            background:#eee;
```
Comment (mbr4nt) [inline hni-textiles-search-result-family.html]: @felipebbtoledo can't use colors directly, gotta use one of the brand color variables. if oyu want a light gray, it's var(--asi-logo-main-type-1)

File: hni-textiles-search-page.html [hunk 4]
```diff
@@ -15,8 +24,13 @@
         }
         
         #list {
+            /*margin-left:25%;
+            padding-left:30px;*/
             margin-left:30px;
         }
+        #list .hni-textiles-search-page:nth-child(even) > *:first-child {
```
Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo if something doesn't work, undo. don't push it here.

File: hni-textiles-search-result-family.html [hunk 5]
```diff
@@ -28,10 +28,15 @@
         poly-grid {
             margin:-10px;
         }
+        div.odd {
```
Comment (mbr4nt) [inline hni-textiles-search-result-family.html]: @felipebbtoledo corsim cornao is here.

File: hni-textiles-facets.html [hunk 6]
```diff
@@ -45,13 +45,46 @@
...
     <template>
 
         <div>
-            <paper-input label="Search..." value="{{keywords}}">
+            <!--<paper-input label="Search..." value="{{keywords}}">
                 <paper-icon-button suffix icon="search" on-click="_search"></paper-icon-button>
-            </paper-input>
+            </paper-input>-->
+            <div class="input-group">
+              <input is="iron-input" id="input" type="text" class="form-control" placeholder="SEARCH" on-click="_search">
```
Comment (mbr4nt) [inline hni-textiles-facets.html]: @felipebbtoledo this input has to have value="{{keywords}}"

File: hni-textiles-facet.html [hunk 7]
```diff
@@ -22,7 +34,7 @@
             <div>{{facet.title}}</div>
             <ul>
                 <template is="dom-repeat" items="{{facet.values}}" as="filter">
-                    <li on-tap="addFilter">{{_formatFilterString(filter)}}</li>
+                    <li on-tap="addFilter"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M8 5v14l11-7z" class="style-scope iron-icon"></path></g></svg>{{_formatFilterString(filter)}}</li>
```
Comment (mbr4nt) [inline hni-textiles-facet.html]: @felipebbtoledo  What is this?

**Comment by felipebbtoledo** [comment]
@mbr4nt it is supposed to be the iron icon. I was able to use the <paper-icon-button icon="expand-less"></paper-icon-button> in other places but this.

File: bower.json [hunk 8]
```diff
@@ -23,7 +23,8 @@
     "asi-configurator-product": "*",
     "webcomponentsjs": "Polymer/webcomponentsjs#0.7.12",
     "polito-enter-viewport-behavior": "*",
-    "polito-quadradiv": "*"
+    "polito-quadradiv": "*",
+    "paper-icon-button": "PolymerElements/paper-icon-button#~1.0.4"
```
Comment (mbr4nt) [inline bower.json]: @felipebbtoledo please undo this change, we already have a dependency for the entire paper-elements package.

File: demo/index.html [hunk 9]
```diff
@@ -4,6 +4,48 @@
...
+            background:#999; 
+            color:#fff;
+            margin-left:18%;
+            margin-bottom:6px;
+        }
+    </style>
+    <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script type="text/javascript">
+         $(document).ready(function() {
+            var offset = 220;
```
Comment (mbr4nt) [inline demo/index.html]: @felipebbtoledo What does this offset represent?

File: hni-textiles-facet.html (hunk 7 above)
Comment (mbr4nt) [inline hni-textiles-facet.html]: @felipebbtoledo Which icon is this one? can't it be referenced by name?

File: demo/index.html (hunk 9 above)
Comment (felipebbtoledo) [inline demo/index.html]: This offset makes the back to top button visible after that amount value of scrolling down.

Comment (mbr4nt) [inline demo/index.html]: got it. how about changing for $(window).height(); so that it's the screen height?

File: hni-textiles-facet.html (hunk 7 above)
Comment (mbr4nt) [inline hni-textiles-facet.html]: @felipebbtoledo I just saw your old comment now. which icon is it? I can help you making it work.

Comment (felipebbtoledo) [inline hni-textiles-facet.html]: I found a simpler way to use the iron icon without all that. It is just '<iron-icon icon="av:play-arrow"></iron-icon>

File: bower.json (hunk 8 above)
Comment (felipebbtoledo) [inline bower.json]: I could not find it there, maybe it has been fixed as I removed the duplicated dependency for the paper-elements.

File: demo/index.html (hunk 9 above)
Comment (felipebbtoledo) [inline demo/index.html]: I'm not the jquery expert guy, if it will work better than it is work, feel free to change it.

Comment (mbr4nt) [inline demo/index.html]: @felipebbtoledo  please change it

File: hni-textiles-facet.html [hunk 10]
```diff
@@ -22,7 +35,7 @@
             <div>{{facet.title}}</div>
             <ul>
                 <template is="dom-repeat" items="{{facet.values}}" as="filter">
-                    <li on-tap="addFilter">{{_formatFilterString(filter)}}</li>
+                    <li on-tap="addFilter"><iron-icon icon="av:play-arrow"></iron-icon>{{_formatFilterString(filter)}}</li>
```
Comment (mbr4nt) [inline hni-textiles-facet.html]: @felipebbtoledo to make this work, go to hni-textiles.html and add an import to "../iron-icons/av-icons.html"

File: demo/index.html (hunk 9 above)
Comment (felipebbtoledo) [inline demo/index.html]: I do not know how to implement it that way.

File: hni-textiles-facet.html (hunk 10 above)
Comment (felipebbtoledo) [inline hni-textiles-facet.html]: Done.

File: demo/index.html (hunk 9 above)
Comment (mbr4nt) [inline demo/index.html]: @felipebbtoledo Instead of var offset = 200; put var offset = $(window).height();

File: bower.json [hunk 11]
```diff
@@ -19,11 +19,7 @@
     "poly-grid": "Allsteel/poly-grid",
     "iron-elements": "PolymerElements/iron-elements#~1.0.3",
     "paper-elements": "PolymerElements/paper-elements#~1.0.5",
-    "mat-dropdown": "ExpandJS/mat-dropdown#~0.9.9",
-    "asi-configurator-product": "*",
-    "webcomponentsjs": "Polymer/webcomponentsjs#0.7.12",
```
Comment (mbr4nt) [inline bower.json]: @felipebbtoledo you can't remove those dependencies, please put them back

Comment (felipebbtoledo) [inline bower.json]: Got it, done.

File: bower.json [hunk 12]
```diff
@@ -21,9 +21,7 @@
     "paper-elements": "PolymerElements/paper-elements#~1.0.5",
     "mat-dropdown": "ExpandJS/mat-dropdown#~0.9.9",
     "asi-configurator-product": "*",
-    "webcomponentsjs": "Polymer/webcomponentsjs#0.7.12",
-    "polito-enter-viewport-behavior": "*",
```
Comment (mbr4nt) [inline bower.json]: @felipebbtoledo Still missing dependencies

Comment (felipebbtoledo) [inline bower.json]: @mbr4nt Done.

File: hni-textiles-search-page.html [hunk 13]
```diff
@@ -31,10 +62,13 @@
                 <hni-textiles-facets facets="{{results.facets}}" filters="{{results.filters}}" keywords="{{keywords}}"></hni-textiles-facets>
             </div>
             <div class="results">
-
+                <div class="pull-right">
+                    <button class="btn btn-primary" type="button"><paper-icon-button icon="search"></paper-icon-button> com search</button>
```
Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo this project can't use bootstrap in the polymers and all colors need to be brand colors coming from the variables, this is rendering blue.

Comment (felipebbtoledo) [inline hni-textiles-search-page.html]: @mbr4nt I did not add bootstrap to the project. I simply reused its class and it worked, so it must be embedded somewhere. Anyway, I added a css class named pull-right to make float right, commited and pushed. Can you specify what is rendering blue? I don't see it. Btw, the com "search and download" buttons have a gradient background color as seen in Brandon's mockup. For achieve that, I had to add a lighter green color to the asi-paper-styles.html. If we can proceed this way, let me know how should we resolve it.

Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo bootstrap is painting it blue:

image

name your classes something else and it should make the blue go away.

As for adding non-brand colors, we can't do that. Can I see what's in Brandon's mockup?

Comment (felipebbtoledo) [inline hni-textiles-search-page.html]: Brandon's mockups are in the folder V11 at https://www.dropbox.com/sh/8a81yu27p51xkpc/AADubyvG_uoCBpccr_0yZzXna?dl=0

**Comment by felipebbtoledo** [comment]
@mbr4nt I replaced the bootstrap css class name for the buttons, so it should be ok now.

File: hni-textiles-search-page.html [hunk 14]
```diff
@@ -10,13 +11,46 @@
...
+        
+        paper-icon-button {
+            height:22px;
+            padding:2px;
+        }
+        
+        .button {
+            /*background-color: var(--asi-paper-primary-accent-color);*/
+            background-image: -webkit-gradient(linear,left top,left bottom,color-stop(0, var(--asi-gradient-1)),color-stop(1, var(--asi-paper-primary-accent-color)));
+            background-image: -o-linear-gradient(bottom, var(--asi-gradient-1) 0%, var(--asi-paper-primary-accent-color) 100%);
```
Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo where are you defining this --asi-gradient-1?

Comment (felipebbtoledo) [inline hni-textiles-search-page.html]: @mbr4nt It is in the same place as the other colors: asi-paper-styles.html

Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo I see. 2 things:
1) nothing you change inside the bower components folder will go through. If you need to change something in asi-paper you need to let me know so I change it in the source code for you. You only modified your copy of it, not the source.
2) you can't add colors to AO. all brand colors are in the styles already, any other color is "illegal"

Comment (felipebbtoledo) [inline hni-textiles-search-page.html]: Ok, what is the agreement we have about the Brandon's mockups color fidelity? Because the "legal" colors are not enough to make it look right.

Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo brand guidelines are more important. Please pick the brand color that best fits. There's a variable for each one already.

Brand Colors
 
 
Berry | PANTONE 683
CMYK: 45/92/40/20
RGB: 130/48/90
HEX: 82305A
 
Cardinal Red | PANTONE 7621C
CMYK: 22/100/85/15
RGB: 174/25/50
HEX: AE1932
 
Spice | PANTONE 7413
CMYK: 7/57/100/00
RGB: 230/133/21
HEX: E68515
 	 	 
 
Pear | PANTONE 383
CMYK: 38/20/100/00
RGB: 168/173/0
HEX: A8AD00
 
Aqua Mint | PANTONE 7472
CMYK: 68/4/38/00
RGB: 69/184/172
HEX: 45B8AC
 
Ocean | PANTONE 7697
CMYK: 78/35/24/00
RGB: 51/136/166
HEX: 3388A6
 	 	 
 
Navy | PANTONE 540
CMYK: 100/77/38/36
RGB: 0/50/91
HEX: 00325B
 
Stone | PANTONE 414
CMYK: 37/29/42/1
RGB: 167/166/149
HEX: A7A695
 
 
Logo & Main Type
 
 
Light Gray | PANTONE 423
CMYK: 47/37/39/3
RGB: 142/144/143
HEX: 8E908F
 
Dark Gray | PANTONE 432
CMYK: 77/63/53/41
RGB: 54/66/74
HEX: 36424A

Comment (felipebbtoledo) [inline hni-textiles-search-page.html]: is it ok to use a legal gray color with opacity? This way we could achieve a lighter tone that matches the desired look and feel.

Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo  yes, that will be fine.

File: demo/index.html [hunk 15]
```diff
@@ -4,6 +4,7 @@
     <script src="../webcomponents-lite.min.js"></script>
     <link rel="import" href="../hni-textiles.html" />
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" />
+    <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
```
Comment (mbr4nt) [inline demo/index.html]: @felipebbtoledo I removed the back to top button from here because I moved it to the asi-paper component. To make it work, please delete your bower_components folder and run a bower install to recreate it. Thanks.

File: hni-textiles-search-page.html (hunk 14 above)
Comment (mbr4nt) [inline hni-textiles-search-page.html]: @felipebbtoledo  I'll merge this as is because i want to send it to QA. This gradient color will be missing. I'll open a bug and assign to you.

File: demo/index.html (hunk 15 above)
Comment (felipebbtoledo) [inline demo/index.html]: Ok, it is done.
